package.json

```diff
@@ -22,6 +22,7 @@
     "gatsby-plugin-styled-components": "^5.8.0",
     "gatsby-source-prismic": "^5.2.5",
     "html-react-parser": "^1.4.9",
+    "prismic-reactjs": "^1.3.4",
```

Comment:
I don't really know if this is a super big deal but in the `{PrismicVideo.uid}.js` page I use the `<PrismicRichText>` component provided by `@prismicio/react` which is already a dependency of the gatsby source plugin. I don't know if there is a difference between the two but I feel like we should only use one for consistency

Reply:
+1 i'd just use `@prismicio/react` when possible

Reply:
switched to @prismicio/react!